analiza-istniejacych-rozwiazan-marketplace.rst

```diff
@@ -0,0 +1,273 @@
+Analiza istniejących rozwiązań marketplace’ów
+=============================================
+
+Streszczenie
+------------
+
+Cel
+~~~
+
+Opracowanie listy istniejących rozwiązań, które można wykorzystać do
+stworzenia marketplace’u aplikacje.gov.pl.
```

Comment:
Nie dopisywałbym "i platformy".

To jest duża zmiana zakresu analizy. W tym dokumencie jest trochę o marketplace, ale o platformie jest mało.